paper-input-container.html

```diff
@@ -369,6 +369,9 @@
     },
 
     ready: function() {
+      if (!this._addons) {
```

Comment:
Yeah, there could be no addons. I can lazily check for `this._addons` where it's referenced if you prefer.